README.md

```diff
@@ -4,7 +4,7 @@ Snipsync makes sure your documented code snippets are always in sync with your G
 
 ## Prerequisites
 
-This tool requires [Node](https://nodejs.org/) and [Yarn](https://yarnpkg.com/).
+This tool requires [Node](https://nodejs.org/) v15.0.0 or above (recommended 15.2.1) and [Yarn](https://yarnpkg.com/).
```

Comment:
i would recommend node 14 if you want maximum ease of use (even numbers are LTS versions) but its nbd at this stage

Reply:
I see. I am not sure it will work with 14. I read that "optional chaining" was still not supported by default in that version.
So, 16 might be our next LTS?